package.json

```diff
@@ -24,20 +26,5 @@
   "bugs": {
     "url": "https://github.com/jillix/engine-keypress/issues"
   },
-  "engine": {
```

Comment:
Can you please add this back? This is used by `blah` to generate the documentation. :books: